cephfs/file_test.go

```diff
@@ -415,3 +415,135 @@ func TestFchown(t *testing.T) {
 	err = f2.Fchown(bob, bob)
 	assert.Error(t, err)
 }
+
+func TestFallocate(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+	useMount(t)
+	fname := "file1.txt"
+	f, err := mount.Open(fname, os.O_RDWR|os.O_CREATE, 0666)
+	assert.NoError(t, err)
+	assert.NotNil(t, f)
+	defer func() {
+		assert.NoError(t, f.Close())
+		assert.NoError(t, mount.Unlink(fname))
+	}()
+
+	// assert that negative values will return error.
+	t.Run("NegativeOffsetLength", func(t *testing.T) {
+		err = f.Fallocate(0, -1, 10)
```

Comment:
Maybe add a `NoFallocFlag` or so that there is no need to pass `0` directly? We have `NoShapshot` as a similar placeholder in the rbd package.